samples/ratelimit/rate-limit-service.yaml

```diff
@@ -113,13 +113,15 @@ spec:
         app: ratelimit
     spec:
       containers:
-      - image: envoyproxy/ratelimit:9d8d70a8 # 2022/08/16
+      - image: envoyproxy/ratelimit:30a4ce1a # 2024/08/01
         imagePullPolicy: IfNotPresent
         name: ratelimit
         command: ["/bin/ratelimit"]
         env:
         - name: LOG_LEVEL
           value: debug
+        - name: DISABLE_STATS
```

Comment:
why disable stats？

Reply:
Sorry not response in time.
This bring bring millions logs. Most custom do not why do not know how to disable stats.
Somehow these logs is useless, compare with debug log_level.

Reply:
https://github.com/envoyproxy/ratelimit/issues/764
FYI